sktime/classification/feature_based/_catch22_classifier.py

```diff
@@ -96,27 +92,31 @@ def __init__(
 
         self._transformer = None
         self._estimator = None
-        self.n_classes = 0
-        self.classes_ = []
+
         super(Catch22Classifier, self).__init__()
 
     def _fit(self, X, y):
-        """Fit an estimator using transformed data from the Catch22 transformer.
+        """Fit a pipeline on cases (X,y), where y is the target variable.
 
         Parameters
         ----------
-        X : nested pandas DataFrame of shape [n_instances, n_dims]
-            Nested dataframe with univariate time-series in cells.
-        y : array-like, shape = [n_instances] The class labels.
+        X : 3D np.array of shape = [n_instances, n_dimensions, series_length]
```

Comment:
Technically the type is np.ndarray but everyone knows what np.array is so isn't an issue.